### DISCUSSION TITLE
想问一下益辉君的头像是咋生成的

### DISCUSSION THREAD

**Discussion by Tianxiang-C** [rbind/yihui#1557] [Q&A] [answered]
如题，在尝试搜索益辉君的个人网站、GitHub、和统计之都论坛之后，没有找到答案，于是特地来这里问一下。

**Comment by yihui** [comment, marked as answer]
关于这个头像，完整的故事在：https://yihui.org/cn/2020/02/manjushri/ 制作它的网站在下面的评论中。

**Reply by Tianxiang-C** [reply]
感谢
